src/aspire/basis/fle_2d.py

```diff
@@ -47,11 +47,6 @@ def __init__(
               be set to that of `FBBasis2D`, and the FLE frequency thresholding procedure to reduce the
               number of functions will not be carried out. This means the number of basis functions for
               a given image size will be identical across the two bases.
-            - The signs of basis functions and coefficients with `sgn == 1` will be flipped relative to
```

Comment:
Since there's only one entry, don't need a list anymore.

Reply:
Sure, sorry I missed that. Removed it and adjusted the text a little.